gitpod-io/website
# :rocket: New `/kubecon-demo` page

## Description

The page would be live on `/kubecon-demo`. And, I will update the same page with a description, calendly URL, etc for every event. And, this would make URLs consistent and easily accessible at every time in the future.

## Related Issue(s)



Fixes #3521

## How to test



- Open [`/kubecon-demo`](https://deploy-preview-3524--gitpod-kumquat.netlify.app/kubecon-demo) & see :eyes:

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3524"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
